<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="onoff_COM">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_COM">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>93</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新COM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="check_onoff">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>2400</width>
      <height>900</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="available_COM">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QComboBox" name="set_BUAD">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>921600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1382400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4000000</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="draw_pid">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>210</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>pid调试</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="draw_buck">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>buck调试</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enter_send">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>450</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>回车发送</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>601</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>180</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>2800</width>
      <height>2100</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="timed_send">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>定时发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="add_crlf">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>添加\r\n</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="draw_dynamic">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>动态绘制(数据直出)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="time_edit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>420</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_data">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="empty_rx">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清接收区</string>
    </property>
   </widget>
   <widget class="QPushButton" name="empty_tx">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清发送区</string>
    </property>
   </widget>
   <widget class="QLabel" name="buffer_size">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>当前缓存：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_data">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>313</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>保存数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_image">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dynamic_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="buck_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="pid_label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hex_send">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>79</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>hex发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
